src/scipp/html/formatting_datagroup_html.py

```diff
@@ -40,24 +74,23 @@ def _summarize_atomic_variable(var, name: str, depth: int = 0) -> str:
     preview = ''
     parent_obj_str = ''
     objtype_str = type(var).__name__
-    if isinstance(var, (Dataset, DataArray, Variable, DataGroup)):
```

Comment:
Why was `DataGroup` removed here?

Reply:
It's because `DataGroup` should be formatted by `_collapsible_summary`, not `_summarize_atomic_variable`.
It's filtered in the `_datagroup_detail`.